.circleci/config.yml

```diff
@@ -53,15 +57,9 @@ jobs:
           command: |
             swagger validate "/go/src/github.com/dragonflyoss/Dragonfly/apis/swagger.yml"
       - run:
-          name: use gometalinter to check gocode of this project.
+          name: use golangci-lint to check gocode of this project.
           command: |
-            make go-mod-vendor
-            gometalinter --disable-all --cyclo-over=20 --skip vendor -E gofmt -E goimports -E golint -E misspell -E vet -E goconst -E gocyclo ./...
-      - run:
-          name: detect deadcode without test folder
-          command: |
-            make go-mod-vendor
-            gometalinter --disable-all --skip vendor --skip test -E deadcode ./...
+            golangci-lint run
```

Comment:
I am wondering if it is a good way to use `make golangci-lint` to do this part. Then this command in Makefile could be used locally on developers' environment. WDYT?

Reply:
Agree. I will update it soon.